mailman/patchset.py

```diff
@@ -19,10 +19,13 @@ def try_or_false(do, exc):
         return False
 
 
-def tag_and_push(repo_path, tag_name):
+def tag_and_push(repo_path, tag_name, msg=None):
     try:
         repo = git.Repo(repo_path)
-        repo.create_tag(tag_name)
+        with repo.config_writer() as config:
+            config.set_value('user', 'name', 'mailman')
+            config.set_value('user', 'email', 'mailman@mailman')
+        repo.create_tag(tag_name, message=msg)
```

Comment:
yeah maybe it is better to push the `git.Repo` out to the containing with block and then pass it instead of the path to the functions so we can do this only once and then unlock some other cleanups (e.g. similar code trying to set the author with git am etc)

Reply:
yeah